models/staging/stg_servicenow__change_request.sql

```diff
@@ -53,8 +52,8 @@ final as (
         on_hold_reason as change_request_on_hold_reason,
         on_hold_task,
         outside_maintenance_schedule,
-        phase as change_request_phase,
-        phase_state as change_request_phase_state,
+        cast(phase as {{ dbt.type_string() }}) as change_request_phase,
```

Comment:
These small cast changes should also be documented in the CHANGELOG. We don't need to be 100% specific to what fields were changed, but we should still make a clear callout that these changes were made.

Reply:
Same comment for similar changes in staging models.

Reply:
updated!